### PR TITLE
Made projection context available when determining the projection key in CrudAction

### PR DESCRIPTION
**Pull request by Jubast**
Hey, It would be really awesome if we could access ProjectionContext in CrudAction.

Any comments would be greatly appreciated :)

### CODE REVIEW

**Comment by dennisdoomen** [comment]
Looks like the build fails for unrelated reasons. I'll check it it out today or tomorrow. 

I'd prefer a little better commit and PR title though

**Comment by Jubast** [comment]
Thanks for the comments i'll watch out in the future.
Sorry for the late response i was on vacation.

Did you get to review the pr?

**Comment by dennisdoomen** [comment]
I just got back from vacation as well ;-)

File: Src/LiquidProjections/MapBuilding/IAction.cs [hunk 1]
```diff
@@ -88,6 +123,18 @@ public interface ICrudAction<TEvent, TProjection, TKey, TContext> : IAction<TEve
...
+        /// <summary>
+        /// Continues configuring a handler for events of type <typeparamref name="TEvent"/>.
+        /// Specifies that the projection with the specified key will be updated by the event handler.
+        /// The event will not be handled by the handler if the projection with such key does not exist.
+        /// </summary>
+        /// <param name="getKey">The delegate that determines the projection key for the event.</param>
+        /// <returns>
+        /// <see cref="IUpdateAction{TEvent,TKey, TProjection,TContext}"/> that allows to continue configuring the handler.
+        /// </returns>
+        [Obsolete("Use AsUpdateOf().IgnoringMissing() instead")]
```
Comment (dennisdoomen) [inline Src/LiquidProjections/MapBuilding/IAction.cs]: Let's not add additional overloads that are obsolete.

Comment (Jubast) [reply Src/LiquidProjections/MapBuilding/IAction.cs]: I agree, will remove 👍

**Comment by dennisdoomen** [comment]
I already did last week, but apparently my NuGet publishing key expired. It just got pushed again.

**Comment by Jubast** [comment]
Thanks 😄

**Pull request merged**